rest-server/src/controllers/job.js

```diff
@@ -54,6 +54,30 @@ const load = (req, res, next, jobName) => {
   });
 };
 
+const init = (req, res, next) => {
+  const jobName = req.body.jobName;
+  new Job(jobName, (job, error) => {
+    if (error) {
+      if (error.message === 'JobNotFound') {
+        req.job = job;
+        next();
+      } else {
+        logger.warn('internal server error');
+        return res.status(500).json({
+          error: 'InternalServerError',
+          message: 'internal server error',
+        });
+      }
+    } else {
+      logger.warn('duplicate job %s', jobName);
+      return res.status(400).json({
+        error: 'DuplicateJobSubmission',
+        message: `job already exists: '${jobName}'`,
+      });
+    }
```

Comment:
If job submit successfully, will this callback not called?

Reply:
callback is called in `if` block, [line 63](https://github.com/Microsoft/pai/pull/398/files#diff-7a88852b42b873aaca542a12ef83bf5bR63)